### DISCUSSION TITLE
How to build the example model in tutorial 1.4

### DISCUSSION THREAD

**Discussion by 2651666L** [gempy-project/gempy#768] [Q&A] [answered]
In the '1.4 Unconformity relationships' tutorial, the part of input data is loading the example model. The rest parts of tutorial are all about the demonstration of the example model but have no process about how to build the unconformity model. Therefore, I copy the source csv data from the example model and try to build by myself. However, I couldn't build a right surface relationships as example model.
My model's 2d image
![image](https://user-images.githubusercontent.com/102043272/223367010-2edecd88-7760-400b-a117-e663058c2c38.png)
Example model's 2d image
![image](https://user-images.githubusercontent.com/102043272/223367472-78a5fa5c-95cf-4a0c-bb2d-f821364bb8cd.png)
Anyone one can help me how to build the example model?

**Comment by AlexanderJuestel** [comment]
Dear @2651666L 
Thank you for your message. 

Indeed, the tutorial 1.4 is not very clear with elaborating on how the model is built. 

Have a look at https://github.com/cgre-aachen/gemgis_data/tree/main/notebooks/05_unconformal_layers

There, I am currently creating notebooks for different structural settings, among them unconformable layers. There you will see that unconformable layers are modeled using different series. The formations within one series are still parallel. 

![image](https://user-images.githubusercontent.com/45469915/223370581-a550b96a-2fe0-4bf5-984e-ca582e37400a.png)

Let me know if that helps in reconstructing the tutorial.

**Reply by 2651666L** [reply]
Well, I had assigned different series. 
Here is my series settings:
![image](https://user-images.githubusercontent.com/102043272/223386120-399c84d6-ede7-4171-90da-69818984d13e.png)
![image](https://user-images.githubusercontent.com/102043272/223386748-2f59456c-bb05-4f30-8519-77e900ebf817.png)
Is there any wrong to my setting?

**Reply by 2651666L** [reply]
Here is another setting, the result remains wrong.
![image](https://user-images.githubusercontent.com/102043272/223416885-54335cfe-5618-4d07-a8ba-9a8d0f3c215c.png)
![image](https://user-images.githubusercontent.com/102043272/223416967-87c3df8e-ffa2-4176-a1fc-b61e90ed65e8.png)
![image](https://user-images.githubusercontent.com/102043272/223417287-40453998-26ed-4de8-bcfa-f6af54db40dc.png)

**Comment by 2651666L** [comment, marked as answer]
I have fixed this problem.

It needs to use the 'geo_model.set_bottom_relation' function to set the series. " geo_model.set_bottom_relation('right','Onlap') " function can make the right series become a onlap relation. Then, the upper series won't cut across the lower series.
![image](https://user-images.githubusercontent.com/102043272/223669379-54b79e1d-b80b-49b7-b346-d1f1f07eb560.png)
